packages/core/test/shared/helpers.ts

```diff
@@ -1,7 +1,106 @@
+import { getDefaultLoggerOptions, pino } from "@walletconnect/logger";
+import { CoreTypes, ICore } from "@walletconnect/types";
+import { DEFAULT_DB_NAME, MOCK_STORE_NAME, TEST_CORE_OPTIONS, storeTestValues } from "./values";
+import { Core, Store } from "../../src";
+
+export type coreStorageData = {
```

Comment:
Types should be capitalized:

```suggestion
export type CoreStorageData = {
```

Reply:
Maybe can remove entirely if `createCoreBackup` in fact isn't used (anymore)